packages/console/src/pages/OrganizationTemplate/index.tsx

```diff
@@ -34,14 +34,15 @@ function OrganizationTemplate() {
   const { getDocumentationUrl } = useDocumentationUrl();
   const [isGuideDrawerOpen, setIsGuideDrawerOpen] = useState(false);
   const {
-    currentSubscription: { planId },
+    currentSubscription: { planId, isEnterprisePlan },
     currentSubscriptionQuota,
   } = useContext(SubscriptionDataContext);
   const { isDevTenant } = useContext(TenantsContext);
+  const isPaidTenant = isPaidPlan(planId, isEnterprisePlan);
+
   const isOrganizationsDisabled =
-    isCloud &&
-    !isFeatureEnabled(currentSubscriptionQuota.organizationsLimit) &&
-    planId !== ReservedPlanId.Pro;
+    isCloud && !isFeatureEnabled(currentSubscriptionQuota.organizationsLimit) && !isPaidTenant;
```

Comment:
This approach is not very straightforward and might benefit from adding comments to clarify.
For paid tenants, we could allow them to enable organization functionality through add-on.

Reply:
added